### PR TITLE
refactor(app): restyle confirm crash recovery panel

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Update confirm crash recovery modal to new designs

Closes [RAUT-160](https://opentrons.atlassian.net/browse/RAUT-160)

# Changelog

- modernize design/localize `ConfirmCrashRecoveryModal` and rename to `ConfirmCrashRecovery`
- update all usage of `useConfirmCrashRecovery`

# Review requests

- with the `enableCalibrationWizards` ff set to true, run through any of health check, deck, pipette, or tip length calibration and click the "Start over" link to the bottom right of any panel with `JogControls`. Clicking "resume" should return to the original panel. Clicking "start over" should return you to the relevant previous step required to try that action again.  

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/CalibrationPanels/ConfirmCrashRecovery.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        width="100%"
+        marginTop={SPACING.spacing4}
+        alignItems={ALIGN_CENTER}
+        justifyContent={JUSTIFY_FLEX_END}
+        gridGap={SPACING.spacing4}
+      >
+        <Link
+          role="button"
+          css={TYPOGRAPHY.darkLinkH4SemiBold}
+          textTransform={TEXT_TRANSFORM_CAPITALIZE}
```
Comment (jerader) [inline app/src/organisms/CalibrationPanels/ConfirmCrashRecovery.tsx]:
```suggestion
          textTransform={TYPOGRAPHY.textTransformCapitalize}
```

**Review by jerader — APPROVED** [review]
lgtm!

File: app/src/organisms/CalibrationPanels/ConfirmCrashRecovery.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  Flex,
+  Link,
+  SPACING,
+  DIRECTION_COLUMN,
+  JUSTIFY_SPACE_BETWEEN,
+  JUSTIFY_FLEX_END,
+  TYPOGRAPHY,
+  ALIGN_CENTER,
+} from '@opentrons/components'
+import { useTranslation } from 'react-i18next'
```
Comment (koji) [inline app/src/organisms/CalibrationPanels/ConfirmCrashRecovery.tsx]: we can move this line after `react`

File: app/src/organisms/CalibrationPanels/useConfirmCrashRecovery.tsx [hunk 3]
```diff
@@ -1,63 +1,55 @@
 import * as React from 'react'
-import { css } from 'styled-components'
-
-import { getLabwareDisplayName } from '@opentrons/shared-data'
+import { useTranslation } from 'react-i18next'
 
 import * as Sessions from '../../redux/sessions'
```
Comment (koji) [inline app/src/organisms/CalibrationPanels/useConfirmCrashRecovery.tsx]: We can move this line after `@opentrons/components`

File: app/src/organisms/CalibrationPanels/useConfirmCrashRecovery.tsx [hunk 4]
```diff
@@ -1,63 +1,55 @@
...
 
 import type { CalibrationPanelProps } from './types'
-
-const CONDITION = 'Jog too far or bend a tip?'
-const START_OVER = 'Start over'
-
-export interface Props extends CalibrationPanelProps {
-  requiresNewTip: boolean
-}
+import { StyledText } from '../../atoms/text'
```
Comment (koji) [inline app/src/organisms/CalibrationPanels/useConfirmCrashRecovery.tsx]: We can move up this line (type should be the last)

File: app/src/organisms/CalibrationPanels/useConfirmCrashRecovery.tsx [hunk 5]
```diff
@@ -1,63 +1,55 @@
...
-      <Text
-        fontSize={FONT_SIZE_BODY_1}
-        color={C_BLUE}
-        as="a"
+    <Flex
+      key="crash-recovery-link"
+      justifyContent={JUSTIFY_CENTER}
+      gridGap={SPACING.spacing2}
+    >
+      <StyledText as="p" fontSize={FONT_SIZE_BODY_1}>
```
Comment (koji) [inline app/src/organisms/CalibrationPanels/useConfirmCrashRecovery.tsx]: Do we still need to use `FONT_SIZE_BODY_1`?

**Review by koji — APPROVED** [review]
Looks good to me!
Left a couple of comments, but they are small things.

**Pull request merged**